sklearn/utils/seq_dataset.pyx.tp

```diff
@@ -1,21 +1,14 @@
-# cython: cdivision=True
-# cython: boundscheck=False
-# cython: wraparound=False
-#
-# Author: Peter Prettenhofer <peter.prettenhofer@gmail.com>
-#
-# License: BSD 3 clause
-
-cimport cython
-from libc.limits cimport INT_MAX
-cimport numpy as np
-import numpy as np
+{{py:
```

Comment:
I don't think it is needed to put those strings inside the `{{py` part (so I would leave the author information and the note you added at the top before the `{{py` stuff)